src/main/java/racing/domain/Winners.java

```diff
@@ -0,0 +1,23 @@
+package racing.domain;
+
+import java.util.List;
+
+public class Winners {
+
+    List<String> winners;
```

Comment:
```suggestion
    private final List<String> winners;
```

불변과 캡슐성을 보장해주면 어떨까요?